.github/workflows/delegator_low-perms_slsa3.yml

```diff
@@ -0,0 +1,279 @@
+# Copyright 2022 SLSA Authors
+#
```

Comment:
Should this still be called generic? `delegator-lowperms_generic_slsa3.yml`, since the workflow is still "generic"? Or we call it `delegator_package_slsa3.yml` since it builds packages

Reply:
lowperms-generic is ok with me